t/rose-ana/01-run-basic-v1.t

```diff
@@ -33,34 +32,34 @@ SUITE_RUN_DIR=$(mktemp -d --tmpdir=$HOME/cylc-run 'rose-test-battery.XXXXXX')
 NAME=$(basename $SUITE_RUN_DIR)
 run_fail "$TEST_KEY" \
     rose suite-run -C $TEST_SOURCE_DIR/$TEST_KEY_BASE --name=$NAME \
-    --no-gcontrol --host=localhost -- --no-detach --debug
+    --host=localhost -- --no-detach --debug
 #-------------------------------------------------------------------------------
 # Test the output
 OUTPUT=$HOME/cylc-run/$NAME/log/job/1/rose_ana_t1/01/job.out
 TEST_KEY=$TEST_KEY_BASE-exact_numeric_success
 file_grep $TEST_KEY "[ OK ].*Semi-major Axis.*all: 0%:" $OUTPUT
 TEST_KEY=$TEST_KEY_BASE-exact_numeric_fail
-file_grep $TEST_KEY "[FAIL].*Orbital Period.*1: 5.234%" $OUTPUT
+file_grep $TEST_KEY "[FAIL].*Orbital Period.*1: 5.234[0-9]*%" $OUTPUT
```

Comment:
Can we please similarly apply a glob or regex pattern to the equivalent line changes in this file, so that other test results do not depend on an exact floating point precision (which would rarely be matched)?

Reply:
Yes. Good idea.